contrib/inventory/nagios_livestatus.py

```diff
@@ -45,7 +47,7 @@
     from mk_livestatus import Socket
 except ImportError:
     print("Error: mk_livestatus is needed. Try something like: pip install python-mk-livestatus")
```

Comment:
replace the print also, 1 is the default return code, also it will send the msg to stderr as it should 

`sys.exit('msg here') `